openedx_events/learning/signals.py

```diff
@@ -136,3 +137,14 @@
         "configuration": CourseDiscussionConfigurationData
     }
 )
+
+# .. event_type: org.openedx.learning.course.persistent_grade.summary.v1
+# .. event_name: PERSISTENT_GRADE_SUMMARY_CHANGED
+# .. event_description: emitted when a grade changes in the course
```

Comment:
can you add something about the persistent nature of the grade?

Reply:
We should add at least a sentence indicating that the event is fired for _persistent_ grades only

Reply:
Yes, you are right, I have fixed it, thanks a lot for the advice.